mender-convert-package

```diff
@@ -223,9 +223,26 @@ rootfsb_end=$(( ${rootfsb_start} + ${rootfs_part_sectors} - 1 ))
 data_start=$(disk_align_sectors ${rootfsb_end} ${MENDER_PARTITION_ALIGNMENT} )
 data_end=$(( ${data_start} + ${data_part_sectors} - 1 ))
 
-# Create partition table. TODO: GPT support
-run_and_log_cmd "${PARTED} -s ${img_path} mklabel msdos"
-run_and_log_cmd "${PARTED} -s ${img_path} unit s mkpart primary fat32 ${boot_part_start} ${boot_part_end}"
+if [ -z "${MENDER_PARTITION_SCHEME}" ]; then
+  # By default use the partition scheme from the input image
+  partition_scheme=$(disk_get_part_value ${disk_image} 1 SCHEME)
+  log_info "Using input partition scheme (${partition_scheme})"
+else
+  # Force partition scheme to the one from configs
+  partition_scheme="${MENDER_PARTITION_SCHEME}"
+  log_info "Forcing partition scheme to ${partition_scheme}"
+fi
```

Comment:
Small nitpick, I would reverse this, e.g

```
if [ -n "${MENDER_PARTITION_SCHEME}" ]; then
  # Force partition scheme to the one from configs
  partition_scheme="${MENDER_PARTITION_SCHEME}"
  log_info "Forcing partition scheme to ${partition_scheme}"
else
# By default use the partition scheme from the input image
  partition_scheme=$(disk_get_part_value ${disk_image} 1 SCHEME)
  log_info "Using input partition scheme (${partition_scheme})"
fi
```

Makes it a bit easier to follow, no functional change. 

Would additionally update the log output to:

```
log_info "Forcing partition scheme defined by MENDER_PARTITION_SCHEME to: ${partition_scheme}"
```

and

```
log_info "Using input diskimage partition scheme (${partition_scheme})"
```

Reply:
If you look in `mender-convert-modify` there are several instances of variables being checked and set this way, so I was just trying to follow those standards.  Basically "if this variable is not set then determine what it should be, otherwise use the set value."  For example `MENDER_DEVICE_TYPE` and `MENDER_GRUB_KERNEL_IMAGETYPE`.  I see your point in wanting the default option to be last though, so I can change it if you prefer.

Is it better to close this pull request and create a new one?  Sorry but I don't use GitHub that often, it seems like if I make a new commit it messes up the existing pull request (i.e. don't need both commits in there).

Reply:
> For example MENDER_DEVICE_TYPE and MENDER_GRUB_KERNEL_IMAGETYPE. I see your point in wanting the default option to be last though, so I can change it if you prefer.

Up to you, a minor thing really.

Reply:
> Is it better to close this pull request and create a new one?

Normally you would just rebase and force push to the same branch. The PR would be updated accordingly.